src/transformers/default-schema-transformer.ts

```diff
@@ -129,6 +130,29 @@ function transformByType(question: FormField) {
         ? 'date'
         : question.questionOptions.rendering;
       break;
+    case 'personAttribute':
+      handlePersonAttributeType(question);
+      break;
+  }
+}
+
+async function handlePersonAttributeType(question: FormField) {
```

Comment:
The asynchronous nature of this function means that the encompassing function (`transform(form: FormSchema)`) should become async.